docs/source/project-configuration.mdx

```diff
@@ -3,142 +3,157 @@ title: Project configuration
+You can link a schema module to every module that includes or consumes your generated operation models.
 
-Use the `.swiftPackageManager` or `.other` options to generate schema types that can be included as their own module.
+The code generation engine can automate the creation of schema modules for the Swift Package Manager. The code generation engine is also flexible enough to support manual configuration for projects that require customized dependencies.
 
-> Most dependency managers like SPM and Cocoapods can automatically include your generated files in the module's directory. This means when generated files are added or removed from the generated module, they will be linked to your project automatically.
+Use the [`.swiftPackageManager`](./code-generation/codegen-configuration#swift-package-manager) or [`.other`](./code-generation/codegen-configuration#other-schema-module-types) options to generate a schema types module.
+
+> Most dependency managers (e.g., SPM and Cocoapods) can automatically include your generated files in the module's directory. This means that when generated files are added or removed from the generated module, they are linked to your project automatically.
 
 #### Embedded in your application
 
-You may also include your schema types directly in a target that you have created.
+You can also include schema types directly in a target that you created.
 
-For this you should use [`ModuleType.embeddedInTarget(name: String)`](./code-generation/codegen-configuration#embedded-in-target) for the [`output.schemaTypes.moduleType`](./code-generation/codegen-configuration#module-type) property. The generated schema types will be enclosed in a caseless namespace enum to prevent naming conflicts with types you may already have defined in your target.
+To do this, use [`ModuleType.embeddedInTarget(name: String)`](./code-generation/codegen-configuration#embedded-in-target) as the value of the [`output.schemaTypes.moduleType`](./code-generation/codegen-configuration#module-type) property. The generated schema types are then enclosed in a caseless namespace enum to prevent naming conflicts with types you already defined in your target.
 
-> By using this option you are responsible for manually adding the generated files to your chosen target. When new generated files are created, or old ones are deleted, you will need to manually add/remove them from your target's "Compile Sources" build phase.
+> This option makes you responsible for **manually** adding generated files to your chosen target. When the code generation engine creates or removes files, you'll need to manually add or remove them from your target's "Compile Sources" build phase.
```

Comment:
> from your target's "Compile Sources" build phase.

I know we had this in the original text but shouldn't we rather tell them to remove them from the project explorer, or whatever that view is referred to as? If we only tell them to remove from "Compile Sources" they may still be in the explorer; for the inexperienced Xcode user. Removing them from the project explorer should handle both steps in one action.

@AnthonyMDev?